packages/playground/website/public/plugin-proxy.php

```diff
@@ -334,6 +334,29 @@ function ($curl, $body) use (&$extra_headers_sent, $default_response_headers) {
         }
 
         $downloader->streamFromGithubReleases($_GET['repo'], $_GET['name']);
+    } else if ( isset( $_GET['branch'] ) ) {
```

Comment:
I'd call it `wordpress-branch` or something to that effect. The word `branch` monopolizes the namespace and poses a question "what parameter would we use for, say, the Gutenberg repository?"